scripts/database/upgrades/upgrade_v4.8.6_to_v4.9.0.sql

```diff
@@ -52,6 +53,8 @@ ALTER TABLE dataset DROP COLUMN globalidcreatetime;
 ALTER TABLE dataset DROP COLUMN identifier;
 ALTER TABLE dataset DROP COLUMN protocol;
 
+ALTER TABLE dvobject DROP COLUMN doiseparator;
+
 --Add new setting into content for shoulder
 INSERT INTO setting(name, content)
 VALUES (':Shoulder', (SELECT substring(content, strpos(content,'/')+1) || '/' from setting  where name = ':Authority'));

```

Comment:
@sekmiller - I'm confused as to why you want to use doiseparator since any value other than '/' would have broken an install at least for >~4.6. Is it for earlier versions? If so, I think you'll need to restore the lines to set the doiseparator in the dvobject table that I removed this morning in my commit. (Sorry for causing confusion...)